### DISCUSSION TITLE
Lock Start date on RangePicker

### DISCUSSION THREAD

**Discussion by webdesign2be** [ant-design/ant-design#45886] [General]
Hi!

Is it possible to lock the start date of a RangePicker after selecting/saving?

I want to be able to show the (already saved) data in the RangePicker and if the user tries to change anything in the Picker, he can only change the end date, but not the start date.

Is that possible in any way?

Thank you in advance!

Kind regards

**Comment by crazyair** [comment]
disabled={[true,false]}

**Reply by webdesign2be** [reply]
Works perfectly, thank you very much!
